DBB Cookbook/Business App Composition Recipes/BusinessApp_Hardware_EOL_EOS_Impact.sql

```diff
@@ -0,0 +1,33 @@
+/*
+ * Business App Hardware EOL/EOS Impacts
+ * This query will joins up with the Compute DBB to gather information on all BusinessApp Impacting Devices
+ * that are going to hit hardware EOL/EOS. Use this if you need to analyze any of the changes needed when that EOL/EOS date approaches.
+ * You can easily shift the window or focus on either EOL/EOS by editing the included WHERE filter clause. 
+ * TIP: Additional information can easily be pulled in from the compute dbb as well. Try using `compute.*` to see what information are available.
```

Comment:
Grammar, "Try using `compute.*` to see what information _is_ available"

Reply:
Also, instead of using "compute.*", would it make more sense to point them to the Data Dictionary entry for Compute_DBB?